integrationtest/src/it/scala/io/gearpump/integrationtest/checklist/RestServiceSpec.scala

```diff
@@ -294,7 +294,27 @@ class RestServiceSpec extends TestSpecBase {
 
   "application life-cycle" should {
     "newly started application should be configured same as the previous one, after restart" in {
+      // setup
+      val originAppId = restClient.submitApp(wordCountJar)
```

Comment:
Better start the application with specified split and sum numbers, e.g. split=4, sum=3. So that we can see, the application is restarted with the same arguments.